client/src/components/Socials/Socials.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import FacebookIcon from '@mui/icons-material/Facebook';
+import TwitterIcon from '@mui/icons-material/Twitter';
+import LinkedInIcon from '@mui/icons-material/LinkedIn';
+import YouTubeIcon from '@mui/icons-material/YouTube';
+import PinterestIcon from '@mui/icons-material/Pinterest';
+import TelegramIcon from '@mui/icons-material/Telegram';
+import {Box, styled} from '@mui/material'
+
+const SocialsContainer = styled((props) => <Box{...props}/>)(() => ({
```

Comment:
you can simplify it:
```suggestion
const SocialsContainer = styled(Box)(() => ({
```